src/index.ts

```diff
@@ -45,7 +45,7 @@ const createGroups = function (Bottleneck, common) {
   });
 };
 
-export function throttling(octokit: Octokit, octokitOptions = {}) {
+export function throttling(octokit: Octokit, octokitOptions: object) {
```

Comment:
This was resulting in a path of execution not covered even though we have this test:
https://github.com/octokit/plugin-throttling.js/blob/054cefa828fe6bf4db3c8962a2f140ecacebee37/test/plugin.test.ts#L42

I think it's because `@octokit/core` is already covering that here:
https://github.com/octokit/core.js/blob/0151ef0f3536441393ab84a4e7d988067ab7ff49/src/index.ts#L73